codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/CodeLayout.kt

```diff
@@ -105,9 +105,9 @@ internal fun composeJavaClassName(
 }
 
 /**
- * Obtains a fully-qualified Java class, generated for the Protobuf type with this name.
+ * Obtains a fully qualified Java class, generated for the Protobuf type with this name.
  */
-internal fun TypeName.javaClassName(declaredIn: File): ClassName =
+internal fun TypeName.javaClassName(declaredIn: ProtoFileHeader): ClassName =
```

Comment:
Let's discuss this signature and those below. They read weird. Maybe, it's not `declaredIn` anymore?